web/src/beta/features/Editor/Map/InspectorPanel/LayerInspector/DataSource/ResourceUrl/index.tsx

```
@@ -0,0 +1,87 @@
+        <TextInput
+          value={value}
+          appearance="readonly"
+          disabled
+          actions={[
+            <IconButton
+              icon="copy"
+              size="small"
+              appearance="simple"
+              iconColor={theme.content.weak}
+              onClick={handleIconClick}
+            />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)